src/docs/references/events/index.md

```diff
@@ -7,26 +7,44 @@ ingress: Reference documentation for Flex marketplace events.
 published: true
 ---
 
-In a Flex marketplace, *events* are a logical representation of activities that
-happen in the marketplace. A single event captures data about what happened with
-a single marketplace object (a user, listing, transaction, etc). Events include,
-for instance, creating, updating or deleting a user or listing, sending a
-message, initiating a transaction, etc.
-
-Flex keeps a history of all marketplace events for 90 days in production
-marketplaces and for 7 days in test and demo marketplaces. Currently, events can
-be queried via the [Integration
-API](https://www.sharetribe.com/api-reference/integration.html#query-events) or
-viewed via [TODO Flex CLI]() with additional ways to access event data coming up
-in the future.
-
-Note that there can be some delay between an action taking place and the
-corresponding event data becoming available in queries.
+In Flex, *events* represent changes in marketplace data resources such
+as listings, users and transactions. An event captures a single change
+in marketplace data, e.g. a user being created or a listing being
+updated. Events can be further analyzed to interpret them as logical
+actions such as a listing being published, a message being sent or a
+user having changed their email address by looking into what were the
+changed data fields.
+
+Events serve two main purposes:
+1. To allow (admin) users to observe changes in marketplace data for
+   auditing purposes. For example, what is the history of all changes
+   for a particular listing.
+2. To allow programmatically reacting to changes and logical actions
+   in a marketplace.
+
+Allowing programs to observe and react to changes enables efficiently
+solving use cases such as synchronizing changes in marketplace data
+into external places, e.g. a CRM, a spreadsheet or a calendar. It also
+allows programs to react to logical actions as part of the marketplace
+flows. For example, setting a field in user metadata can trigger an
+integration to publish the user's listings.
+
+Currently, Flex exposes events by allowing them to be queried via the
+[Integration
+API](https://www.sharetribe.com/api-reference/integration.html#query-events)
+or viewed via [TODO Flex CLI](). Integration API supports implementing
+efficient polling where only events that have happened since last poll
+query are returned. This makes it possible to keep the polling
+interval short enough to react to events shortly after they occur.
+
+Flex does not retain event data forever. Flex maintains a history of
+all marketplace events for 90 days in production marketplaces and for
+7 days in test and demo marketplaces.
 
 ## Event data
 
-The event object contains data about the event itself, as well as about the Flex
-Integration API resouce for which the event is about (`listing`, `user`,
+The event object contains data about the event itself, as well as
+about the Flex resource which the event is about (`listing`, `user`,
```

Comment:
Should we still emphasize more somewhere that events follow the Integration API resources (and not, say, the Marketplace API resources)? That was my intention here and where using the `INTEGRATION_API_RESOURCE_TYPE/EVENT_SUBTYPE` notation. Or do you see that as inaccurate?

Reply:
@bladealslayer If you remember the "data models of Flex" diagram we sketched, the idea there was that Integration API exposes a canonical resource model. So my intention here was to reduce the prominence of Integration API as the driver for the data model and rather say that this is our default data model, which is exposed in events and by the Integration API.

Reply:
I guessed right that this was the reason then :) I guess my worry is that from reader's perspective, they don't yet have much to point them in that direction. It seemed to me easiest to give the Integration API as a reference point, as this is the only canonical data model they see. Maybe we'll gradually build that picture from now on with events, webhooks, etc.